lab-dc/Makefile

```diff
@@ -3,8 +3,7 @@ build:
 	@cp ./netbox-docker/docker-compose.yml ./netbox-docker/docker-compose.yml.bak
 	@sed -i.bak 's/VERSION-v4.1-3.0.2/VERSION-v3.7/g' ./netbox-docker/docker-compose.yml
 	@cp ./docker-compose.yml ./netbox-docker/docker-compose.override.yml
-	@cd ./netbox-docker && docker-compose build
- # --no-cache
+	@cd ./netbox-docker && docker-compose build --no-cache
```

Comment:
Do we really need to rebuild the images from scratch?

Reply:
I'll remove the folder later, juste ignore it